plugins/pci-bcr/fu-plugin-pci-bcr.c

```diff
@@ -42,6 +52,14 @@ fu_plugin_device_registered (FuPlugin *plugin, FuDevice *dev)
 			priv->bcr_addr = tmp;
 		}
 	}
+	if (g_strcmp0 (fu_device_get_plugin (dev), "flashrom") == 0) {
```

Comment:
is this going to inadvertanely affect UEFI based `main-system-firmware` too later very easily by accident?  That's not flashable from _within_ OS, but absolutely should still be marked updatable for handling the UEFI capsule.

I know right now you're checking for if a flashrom device is modified, but I think you need to check the _specific_ flashrom device is main-system-firmware.

Reply:
> but I think you need to check the specific flashrom device is main-system-firmware

I think we only support one flashrom device right now, and that unconditionally has `but I think you need to check the specific flashrom device is main-system-firmware` but I do get the point. What about something like this:

    -       if (g_strcmp0 (fu_device_get_plugin (dev), "flashrom") == 0) {
    +       if (g_strcmp0 (fu_device_get_plugin (dev), "flashrom") == 0 &&
    +           fu_device_has_instance_id (dev, "main-system-firmware")) {

Reply:
>I think we only support one flashrom device right now, and that unconditionally has main-system-firmware

Right.  I'm just looking out for a future non-obvious bug.

>What about something like this:

Yes - exactly what I had in mind.

Reply:
Okay, fixed, thanks.